### PR TITLE
Fix quantity of blank lines after code object.

### PR DESCRIPTION
**Pull request by stepshal**

### CODE REVIEW

**Comment by jaraco** [comment]
This seems like a reasonable change. Consistency is good. The changes to files in _vendor while acceptable are discouraged as they will be overridden when that package is refreshed and also, it makes the file checksums not match any version in the upstream package. The best thing to do for those would be to request these changes in the relevant upstream package and then they'll eventually get refreshed here.

File: setuptools/unicode_utils.py [hunk 1]
```diff
@@ -4,6 +4,8 @@
 from setuptools.extern import six
 
 # HFS Plus uses decomposed UTF-8
+
```
Comment (jaraco) [inline setuptools/unicode_utils.py]: I think this change is wrong. In my experience, the comments should be adjacent to their relevant code. These additional new lines make the comment appear as a standalone construct.

File: setuptools/tests/test_bdist_egg.py [hunk 2]
```diff
@@ -26,6 +27,7 @@ def setup_context(tmpdir):
 
 
 class Test:
+
```
Comment (jaraco) [inline setuptools/tests/test_bdist_egg.py]: Is the double new line after a class decoration something that's part of the standard? I'm not familiar with it. Can you provide a reference?

File: setuptools/package_index.py [hunk 3]
```diff
@@ -160,6 +163,8 @@ def interpret_distro_name(
         )
 
 # From Python 2.7 docs
+
```
Comment (jaraco) [inline setuptools/package_index.py]: Here again, the whitespace betrays the meaning.

File: pkg_resources/_vendor/pyparsing.py [hunk 4]
```diff
@@ -3162,6 +3233,8 @@ def z(*paArgs):
 #
 # global helpers
 #
+
```
Comment (jaraco) [inline pkg_resources/_vendor/pyparsing.py]: Here's a case where maybe the extra space makes sense. The comment isn't pertinent to the immediate following block but to the rest of the file.

File: pkg_resources/_vendor/pyparsing.py [hunk 5]
```diff
@@ -766,11 +794,14 @@ def nullDebugAction(*args):
 
 # this version is Python 2.x-3.x cross-compatible
 'decorator to trim function calls to match the arity of the target'
+
```
Comment (jaraco) [inline pkg_resources/_vendor/pyparsing.py]: I'm pretty sure this string should be the do string for the function. Perhaps also the comment should be moved inside.

File: setuptools/tests/test_bdist_egg.py (hunk 2 above)
Comment (stepshal) [inline setuptools/tests/test_bdist_egg.py]: it's E309, but it is not included in standard of pep8.

**Comment by stepshal** [comment]
@jaraco Done.

**Comment by stepshal** [comment]
@jaraco Done.

**Comment by stepshal** [comment]
@jaraco In this (https://github.com/pypa/setuptools/pull/648) PR those two line adjacent to their relevant code.

**Pull request merged**